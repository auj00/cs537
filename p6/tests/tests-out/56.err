Traceback (most recent call last):
  File "/home/yashd/private/CS537_Team/cs537/p6/tests/./read-write.py", line 32, in <module>
    contents = file.read()
OSError: [Errno 38] Function not implemented
cat: mnt/file1: Function not implemented
diff: mnt/file1: Function not implemented
