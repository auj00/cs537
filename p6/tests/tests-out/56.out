Correct
file1 readback does not match data written
total allocated datablocks on all disks: found 13 expected either 12 or 14.
